docs/global.css

```diff
@@ -37,3 +37,10 @@
 :global pre {
   font-size: 0.95em;
 }
+
+figcaption {
```

Comment:
This won't do anything as it's targeting an html element, I believe you want the class?

Reply:
No that is actually the intention! This is a real tag apparently, and Guillaume used it in the actual markdown files.

Reply:
(I removed it before, that's why it messed up the spacing)

Reply:
My god, you're right.